### ISSUE TITLE
Add Vivaldi Snapshot and Vivaldi Sopranos to the list of supported browsers

### ISSUE THREAD

**Issue by contribucious** [AdguardTeam/AdguardForAndroid#3400]
[Vivaldi](https://play.google.com/store/apps/details?id=com.vivaldi.browser.snapshot) browser _snapshot_ version was released on the Play Store on September 20, 2019.
Please add it to the list of supported browsers.

**Package name:** com.vivaldi.browser.snapshot

### Thanks in advance!

---
**Related:** https://github.com/AdguardTeam/AdguardForAndroid/issues/3061 (_beta_ version, not the same)
**My AG version used:** 3.3.2 (230) [stable] _(but [Nightly changelog](https://adguard.com/en/versions/android/nightly.html) checked before posting.)_

**Comment by contribucious** [comment]
Consider adding **com.vivaldi.browser.sopranos** too:

> Note that there now exist three different versions of Vivaldi Android: **Beta**, **Snapshot** and **Soprano's**. The latter being pre-snapshot build for translators and Vivaldi team members.
>
> **Source:** [A Vivaldi translator](https://forum.vivaldi.net/topic/41642/bitwarden-v2-2-6-support-vivaldi-android/13)

> Our testing process is pretty straightforward. When we finish a new feature, we first test it internally (with the help of a small group of Vivaldi volunteers called the **Sopranos**), then we release the new feature in a **Snapshot**.
>
> **Source:** [Vivaldi](https://vivaldi.com/blog/join-the-ride/)

**Comment by contribucious** [comment]
Yay! Freshly [added](https://adguard.com/en/versions/android/nightly.html)!
I will close this issue therefore.
Thanks again to the team! :thumbsup:

> ## AdGuard for Android
> ### version 3.4 nightly 18 from April 21, 2020
> [...]
> [Other] Add Vivaldi Snapshot & Sopranos to the list of supported browsers: #3400
> [...]

**Comment by contribucious** [comment]
@Revertron Feel free to add the `Resolution: Done` (or `Resolution: Fixed`) tag as needed, however. :thumbsup:

**Comment by contribucious** [comment]
@Revertron 

According to my tests, you forgot to add `com.vivaldi.browser.snapshot` and `com.vivaldi.browser.sopranos` to `pref.enforce.https.filtering` to have HTTPS filtering active by default.

Tested OK once done manually (either by adding them manually to the `pref.enforce.https.filtering` list or by activating their option "HTTPS filtering" in the interface).

Can you confirm that? And if so, correct that? Thank you in advance. :thumbsup:

___
**Environment:** Passage from the last stable version (3.3.3) to the last beta version (3.4 beta 1) — not premium.

**Comment by contribucious** [comment]
### Regarding the post above:
&nbsp;
Unlike [Brave Beta](https://github.com/AdguardTeam/AdguardForAndroid/issues/3401#issuecomment-631088806), _NOT_ fixed in `3.4 beta 2` in this case.
&nbsp;
```console
$ grep -n vivaldi adguard_3.4_beta-1_apk_extracted/enforce_https_filtering_apps.txt
33:com.vivaldi.browser
```
```console
$ grep -n vivaldi adguard_3.4_beta-2_apk_extracted/enforce_https_filtering_apps.txt
33:com.vivaldi.browser
```
&nbsp;
(cc @admitrevskiy Thanks in advance.)

**Comment by contribucious** [comment]
@artemiv4nov Hello!

> According to my tests, you forgot to add `com.vivaldi.browser.snapshot` and `com.vivaldi.browser.sopranos` to `pref.enforce.https.filtering` to have HTTPS filtering active by default.
>
> Tested OK once done manually (either by adding them manually to the `pref.enforce.https.filtering` list or by activating their option "HTTPS filtering" in the interface).

:arrow_right_hook: Could you remedy this before [beta 1](https://github.com/AdguardTeam/AdguardForAndroid/issues/3531)? :thumbsup:

:arrow_right: More generally, is there a particular reason why some little-known browsers (in beta, dev, etc.) are added to this list while more well-known ones (same type of version) are not?

_Thank you in advance!_
